packages/inventories-exporter/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@commercetools/inventories-exporter",
+  "version": "1.0.0-development",
+  "description": "Export inventories from the commercetools platform.",
+  "keywords": [
+    "commercetools",
+    "inventories",
+    "stock",
+    "code",
+    "export",
+    "exporter"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs/tree/master/packages/inventories-exporter"
+  },
+  "homepage": "https://commercetools.github.io/nodejs/cli/inventories-exporter.html",
+  "bugs": "https://github.com/commercetools/nodejs/issues",
+  "license": "MIT",
+  "author": {
+    "name": "Abimbola Idowu",
+    "email": "abimbola.idowu@commercetools.de",
+    "url": "hisabimbola.com"
+  },
+  "bin": {
+    "inventoriesexporter": "bin/inventories-exporter.js"
+  },
+  "main": "lib/index.js",
+  "files": [
+    "lib",
+    "bin"
+  ],
+  "dependencies": {
+    "@commercetools/api-request-builder": "^3.1.1",
+    "@commercetools/get-credentials": "^2.1.0",
+    "@commercetools/sdk-client": "^1.4.1",
+    "@commercetools/sdk-middleware-auth": "^3.1.0",
+    "@commercetools/sdk-middleware-http": "^2.1.1",
+    "@commercetools/sdk-middleware-user-agent": "^1.2.0",
+    "JSONStream": "^1.2.1",
+    "fast-csv": "^2.4.0",
+    "lodash.memoize": "^4.1.2",
+    "npmlog": "^4.1.2",
+    "pretty-error": "^2.1.1",
+    "yargs": "^8.0.2"
+  },
+  "devDependencies": {
```

Comment:
Shouldn't this be in the toot `package.json` file? Despite the extraneous dependency error?